src/Components/animation/AnimationEditor.js

```diff
@@ -310,7 +374,10 @@ class AnimationEditor extends React.Component {
 
     if (deType === 'triangle') key = 602;
 
-    if (deType === 'disc') key = 603;
+    if (deType === 'disc') {
```

Comment:
`deType` is a bit confusing, could we rename it to something comre meaningful ?

For example `displayedElementType` ? (if it is what de means)

Reply:
Done